ext/string_metric/levenshtein/iterative_with_two_matrix_rows_ext.c

```diff
@@ -0,0 +1,99 @@
+#include <ruby.h>
+
+#define MIN(a, b, c) ((a) < (b) ? ((a) < (c) ? (a) : (c)) : ((b) < (c) ? (b) : (c)))
```

Comment:
The above line could be made much easier to reason with, using nested macros:
    #define MIN2(a, b) (((a) < (b)) ? (a) : (b))
    #define MIN3(a, b, c) (MIN2(MIN2((a), (b)), (c)))